docs/source/stubtest.rst

```diff
@@ -122,14 +122,29 @@ The rest of this section documents the command line interface of stubtest.
     allowlists. Allowlists can be created with --generate-allowlist. Allowlists
     support regular expressions.
 
+    The presence of an entry in the allowlist means stubtest will not generate
+    any errors for the corresponding definition.
+
 .. option:: --generate-allowlist
 
     Print an allowlist (to stdout) to be used with --allowlist
 
+    When introducing stubtest to an existing project, this is an easy way to
+    silence all existing errors.
+
 .. option:: --ignore-unused-allowlist
 
     Ignore unused allowlist entries
 
+    By default, stubtest will complain if an allowlist entry is not necessary
+    for stubtest to pass successfully.
```

Comment:
Maybe make it slightly clearer that "the default" here refers to "the default without this option", not "the default with this option"

```suggestion
    Without this option enabled, the default is for stubtest to complain if an
    allowlist entry is not necessary for stubtest to pass successfully.
```